aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1730,6 +1781,34 @@ impl VMAdapter for AptosVM {
             },
         })
     }
+
+    fn execute_single_transaction_sequential(
+        &self,
+        txn: &PreprocessedTransaction,
+        view: &impl StateView,
+        log_context: &AdapterLogSchema,
+    ) -> Result<(VMStatus, VMOutput, Option<String>), VMStatus> {
+        match self.execute_single_transaction(txn, &view, log_context, true) {
+            Ok((vm_status, vm_output, sender)) => match vm_output.try_materialize(view) {
+                Ok(vm_output) => {
+                    return Ok((vm_status, vm_output, sender));
+                },
+                Err(vm_status) => {
+                    if matches!(vm_status, VMStatus::MoveAbort(_, code) if code == EADD_OVERFLOW || code == ESUB_UNDERFLOW)
```

Comment:
Let's move this to aptos-aggregator? You had a nice function before (`is_aggregator_error`) which is better to use then duplicate the code. This way you also do not have to make `EADD_OVERFLOW` and the other error public.

Reply:
delta_change_set.rs uses PartialVMError when returning errors. It doesn't know about VMStatus, which is a couple of levels higher in the hierarchy we have right now. That's the reason, I earlier had `is_aggregator_error` function VMStatus implementation. Since that isn't an option, I felt it's better to have this code here.

Reply:
Making internal error codes public and leaking what is an aggregator error to VM seems worse to me than using a public struct😄 What is the motivation behind delta_change_set.rs not knowing about VMStatus? It's just an import?